source/appendices/e-audit-requirements-and-tests/07-ir-tests.rst

```diff
@@ -0,0 +1,32 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2021 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+E.7 Item Report Audit Tests
+---------------------------
+
+This section of the appendix outlines tests that MUST be run during audits for a Host_Type that is required to offer the Item Report.
+
+The Item Report will be COUNTER-compliant if the following audit tests are passed. The Standard Views of the Item Report (IR_A1 and IR_M1) will be COUNTER-compliant if the metrics match those in the Item Report.
+
+Item Report audit tests apply to Host_Types Data_Repository, Multimedia, Repository, and Scholarly_Collaboration_Network. Auditors SHOULD also run Item Report audit tests on other Host_Types that have opted to provide Item Reports.
+
+In order for the Item Report to be accurately audited, the report provider MUST supply the auditor with a list of Data_Types represented on the platform.
+
+Note that because Components are optional for Release 5.1, they SHOULD be omitted from Item Report audit tests.
+
+
+E.7.1 Total_Item_Investigations and Unique_Item_Investigations
```

Comment:
Reverse order of this and the next section and offer an exception if all investigations also result in requests (as for the journal tests)?